src/main/java/com/mongodb/hibernate/dialect/MongoDialect.java

```diff
@@ -119,7 +118,7 @@ public void contribute(TypeContributions typeContributions, ServiceRegistry serv
     private void contributeObjectIdType(TypeContributions typeContributions) {
         typeContributions.contributeJavaType(ObjectIdJavaType.INSTANCE);
         typeContributions.contributeJdbcType(ObjectIdJdbcType.INSTANCE);
-        var objectIdTypeCode = MqlType.OBJECT_ID.getVendorTypeNumber();
+        var objectIdTypeCode = ObjectIdJdbcType.MQL_TYPE.getVendorTypeNumber();
```

Comment:
`MqlType` was made package-access. Furthermore, `ObjectIdJdbcType.MQL_TYPE` explicitly points to our implementation of the `JdbcType` for `ObjectId`, which improves readability.